main.js

```diff
@@ -0,0 +1,108 @@
+const button = document.getElementById("button");
+button.addEventListener("click", showDataButton);
+const input = document.getElementById("input");
+input.addEventListener("keyup", showDataKeyboard);
+const page= document.getElementById("mainpage");
+
+function returnURL() {
+    let apiURL = "https://restcountries.eu/rest/v2/name/" + input.value + "?fullText=true";
+    return apiURL;
+}
+
+async function showDataButton(e) {
+        showResults();
+}
+
+function showDataKeyboard(e) {
+    if (e.keyCode === 13) {
```

Comment:
`keyCode` is deprecated. Het wordt je dus afgeraden om dit te gebruiken. Gebruik i.p.v. daarvan `key`.
Bron: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode